### PR TITLE
multisig docs

### PR DESCRIPTION
**Pull request by ozgunozerk**
Multisig documentation

### CODE REVIEW

**Comment by ozgunozerk** [comment]
Will put to chatgpt before merging, do not spend much time on the English :)

**Comment by ozgunozerk** [comment]
Haha sure! Used grammarly this time

File: docs/modules/ROOT/pages/pallets/multisig.adoc [hunk 1]
```diff
@@ -40,13 +40,13 @@ pub fn approve_as_multi<T: Config>(
 ----
 Register approval for a dispatch to be made from a deterministic composite account.
 
-Since the first register (from origin) count as a vote as well, only `threshold - 1` additional votes are necessary from `other_signatories`.
+Since the first register (from origin) counts as a vote as well, only `threshold - 1` additional votes are necessary from `other_signatories`.
 
-Payment: `DepositBase` will be reserved if this is the first approval, plus `threshold` times `DepositFactor`. It is returned once this dispatch happens or is cancelled.
+Payment: `DepositBase` will be reserved if this is the first approval, plus `threshold` times `DepositFactor`. It is returned once this dispatch happens or is canceled.
```
[the remark this answers is not in the document]
Comment (ozgunozerk) [reply docs/modules/ROOT/pages/pallets/multisig.adoc]: makes sense, I'll switch to `cancelled`, I think it's 90% chance that OZ is following American English

File: docs/modules/ROOT/pages/pallets/multisig.adoc [hunk 2]
```diff
@@ -0,0 +1,248 @@
...
+:highlightjs-languages: rust
+:github-icon: pass:[<svg class="icon"><use href="#github-icon"/></svg>]
+
+= Pallet Name
+
+Branch/Release: `release-polkadot-v{1.3.0}`
+
+== Purpose
+
+This module enables multi-signature operations in your runtime. It allows multiple signed origins (accounts) to coordinate and dispatch a call. For the call to execute, the threshold number of accounts from the set (signatories) must approve it.
```
[the remark this answers is not in the document]
Comment (ozgunozerk) [reply docs/modules/ROOT/pages/pallets/multisig.adoc:11]: > Are all signed origins accounts? In other words does there exist any signed origin that is not an account?

Yes, all signed origins are accounts, and no, there does not exist any signed origin that is not an account. 

The reason for this: `It allows multiple signed origins (accounts)` was to make it clear to end developer that, the parameter `origin` represents `accounts` in this context. I think the explanation should stay as is, but if you have a strong preference on committing your suggestion, we can do so 👍

[the remark this answers is not in the document]
Comment (ozgunozerk) [reply docs/modules/ROOT/pages/pallets/multisig.adoc:11]: > And is a multisig an account? So can a multisig be one of the accounts in another multisig?

`multisig` refers to the operation itself (along with threshold, accounts, call, everything). But if you meant to ask, can a multiple account be a part of another multisig operation, then the answer is yes!

So, a multisig account: X -> composed of A,B,C
where A is also a multisig account, and composed of: A1, A2, A3
is completely possible :)

**Review by KitHat — CHANGES_REQUESTED** [review]
Two small comments, in other things, LGTM

File: docs/modules/ROOT/pages/pallets/multisig.adoc [hunk 3]
```diff
@@ -0,0 +1,248 @@
+:source-highlighter: highlight.js
+:highlightjs-languages: rust
+:github-icon: pass:[<svg class="icon"><use href="#github-icon"/></svg>]
+
+= Pallet Name
```
Comment (KitHat) [inline docs/modules/ROOT/pages/pallets/multisig.adoc:5]: Please, update the pallet name 🙂

File: docs/modules/ROOT/pages/pallets/multisig.adoc [hunk 4]
```diff
@@ -0,0 +1,248 @@
...
+
+Branch/Release: `release-polkadot-v{1.3.0}`
```
[the remark this answers is not in the document]
Comment (ozgunozerk) [reply docs/modules/ROOT/pages/pallets/multisig.adoc:7]: ah yeah, that was the from template... Fixing

**Review by KitHat — APPROVED** [review]
Lgtm

**Pull request merged**